packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -1199,6 +1199,53 @@ - (void)setZoomLevel:(CGFloat)zoom withCompletion:(void (^)(FlutterError *_Nulla
   completion(nil);
 }
 
+- (void)setVideoStabilizationMode:(FCPPlatformVideoStabilizationMode)mode
+                   withCompletion:(void (^)(FlutterError *_Nullable))completion {
+  AVCaptureVideoStabilizationMode stabilizationMode = getAvCaptureVideoStabilizationMode(mode);
+
+  if (![_captureDevice.activeFormat isVideoStabilizationModeSupported:stabilizationMode]) {
+    completion([FlutterError errorWithCode:@"VIDEO_STABILIIZATION_ERROR"
+                                   message:@"Unavailable video stabilization mode."
+                                   details:nil]);
+    return;
+  }
+
+  AVCaptureConnection *connection = [_captureVideoOutput connectionWithMediaType:AVMediaTypeVideo];
+
+  connection.preferredVideoStabilizationMode = stabilizationMode;
+
+  completion(nil);
+}
+
+- (BOOL)isVideoStabilizationModeSupported:(FCPPlatformVideoStabilizationMode)mode {
+  AVCaptureVideoStabilizationMode stabilizationMode = getAvCaptureVideoStabilizationMode(mode);
+  return [_captureDevice.activeFormat isVideoStabilizationModeSupported:stabilizationMode];
+}
+
+
+AVCaptureVideoStabilizationMode getAvCaptureVideoStabilizationMode(
```

Comment:
i remember there was a file that contains all similar helpers? Can you move this into that file?

Reply:
Hi, just added a new commit with the requested change. Moved the function into CameraProperties.